src/Renci.SshNet/IPrivateKeyFile.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Renci.SshNet.Security;
+
+namespace Renci.SshNet
+{
+    /// <summary>
+    /// Represents private key file interface.
+    /// </summary>
+    public interface IPrivateKeyFile
```

Comment:
There's nothing file-related in this interface. Do we want to extend it with file related properties or methods, or change the name of the interface?

Reply:
maybe IPrivateKeySource?